src/internal/observable/fromEventPattern.ts

```diff
@@ -137,32 +145,19 @@ export function fromEventPattern<T>(addHandler: (handler: NodeEventHandler) => a
  * @owner Observable
  */
 
-export function fromEventPattern<T>(addHandler: (handler: NodeEventHandler) => any,
-                                    removeHandler?: (handler: NodeEventHandler, signal?: any) => void,
-                                    resultSelector?: (...args: any[]) => T): Observable<T | T[]> {
-
+export function fromEventPattern<T>(
+  addHandler: (handler: NodeEventHandler) => any,
+  removeHandler?: (handler: NodeEventHandler, signal?: any) => void,
+  resultSelector?: (...args: any[]) => T
+): Observable<T | T[]> {
   if (resultSelector) {
     // DEPRECATED PATH
-    return fromEventPattern<T>(addHandler, removeHandler).pipe(
-      mapOneOrManyArgs(resultSelector)
-    );
+    return fromEventPattern<T>(addHandler, removeHandler).pipe(mapOneOrManyArgs(resultSelector));
   }
 
-  return new Observable<T | T[]>(subscriber => {
+  return new Observable<T | T[]>((subscriber) => {
     const handler = (...e: T[]) => subscriber.next(e.length === 1 ? e[0] : e);
-
-    let retValue: any;
-    try {
```

Comment:
Actual try/catch that was removed. The rest of the changes are formatting changes.